src/ui/units/connections/store/actions/index.ts

```diff
@@ -261,13 +264,17 @@ export function createConnection(name: string, dirPath?: string) {
             flow([resetFormsData, dispatch])();
         }
 
-        batch(() => {
-            if (templateFolderId) {
+        if (templateFolderId) {
```

Comment:
This will not work with workbook templates

Reply:
agree, fixed